jenkinsherder.ini

```diff
@@ -1,10 +1,2 @@
-[Credentials]
-key = c3fd801c-a060-4b13-803a-8366463bf0c2
-secret = 424cfa01-a68d-475f-b830-3e3c94b816dd
-
-[Repo]
-project = qa-try
-host = treeherder-dev.allizom.org
-
```

Comment:
All this got moved to `treeherder_config.py` which in turn refers to credentials files.